usaspending_api/api_contracts/contracts/v2/bulk_download/awards.md

```diff
@@ -121,6 +118,10 @@ This route sends a request to the backend to begin generating a zipfile of award
 + `recipient_locations` (optional, array[Location], fixed-type)
 + `sub_agency` (optional, string)
     Sub-agency name to include (based on the agency filter)
++ `sub_award_types` (optional, array[enum[string]])
+    + Members
+        + `grant`
+        + `procurement`
```

Comment:
So... like... my OCD really wants these to be plural like their prime counterparts... and for `procurements` to be called `contracts` for the same reason... and that's (sort of) what it's called on the checkbox on the website... but it's just a suggestion.  Not going to try to hold up the ticket over it.

Reply:
Ugh.  But it's what it's called in the database.  I give up.

Reply:
Reading your comments summed up my internal struggle working on this ticket. I want them to match across the board, but it makes this ticket much more difficult.